src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -24,6 +25,7 @@ private Bugsnag() {}
      */
     public static Client init(Context androidContext) {
         client = new Client(androidContext);
+        configureClientObservers();
```

Comment:
We use Bugsnag entirely without the `Bugsnag` class so :+1: to this and your greater thoughts about not coupling any functionality to this class.